libnopegl/src/nodes.c

```diff
@@ -291,53 +291,51 @@ static int node_init(struct ngl_node *node)
     return 0;
 }
 
-static int node_set_ctx(struct ngl_node *node, struct ngl_ctx *ctx, struct ngl_ctx *pctx)
+static int node_set_ctx(struct ngl_node *node, struct ngl_ctx *ctx)
```

Comment:
Regarding the commit msg:  `nodes: split node_set_ctx() into two separate function`

Reply:
Added `()` and the `s`